src/components/FormField/FormField.tsx

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
-class FormField extends React.Component {
+interface FormFieldProps {
+  id: string;
+  label: string;
+  type: string;
+  fieldName: string;
+  errorState?: any;
+  value: any;
```

Comment:
Value `any`...?

Reply:
This one was interesting. I tried a few different values but what the parent component was sending through was different for a few instances. This `FormField` will most likely go away in a future PR when another React form component is used (react hook forms or formik).